src/common/colorspaces.h

```diff
@@ -70,7 +70,8 @@ typedef enum dt_colorspaces_color_profile_type_t
   DT_COLORSPACE_EXPORT = 16, // export and softproof are categories and will return NULL with dt_colorspaces_get_profile()
   DT_COLORSPACE_SOFTPROOF = 17,
   DT_COLORSPACE_WORK = 18,
-  DT_COLORSPACE_LAST = 19
+  DT_COLORSPACE_LAST = 19,
```

Comment:
DT_COLORSPACE_LAST must remain last, it should be:

DT_COLORSPACE_REC709 = 19,
DT_COLORSPACE_LAST = 20